external/llvm-project/mlir/lib/Dialect/StandardOps/IR/Ops.cpp

```diff
@@ -820,6 +820,23 @@ static LogicalResult verify(ConstantOp &op) {
     return op.emitOpError() << "requires attribute's type (" << value.getType()
                             << ") to match op's return type (" << type << ")";
 
+  if (auto intAttr = value.dyn_cast<IntegerAttr>()) {
```

Comment:
I don't know why this section and the next weren't in the upstream code, but we had simple attributes barfing without them.